test/modules/logicModule/LM_PC_FundingPot_v1.t.sol

```diff
@@ -95,16 +90,1197 @@ contract LM_PC_FundingPot_v1_Test is ModuleTest {
+            _helper_createAccessCriteria(accessCriteriaEnum);
+
+        vm.expectRevert();
+        fundingPot.editAccessCriteriaForRound(
+            roundId, accessCriteriaId, accessCriteria
+        );
+    }
+
+    function testFuzzEditAccessCriteriaForRound_revertsGivenRoundIsActive(
+        uint8 accessCriteriaEnum
+    ) public {
+        vm.assume(accessCriteriaEnum >= 1 && accessCriteriaEnum <= 4);
+
+        _helper_setupRoundWithAccessCriteria(accessCriteriaEnum);
+        uint64 roundId = fundingPot.getRoundCount();
+        uint8 accessCriteriaId = 0;
+
+        // Warp to make the round active
+        (uint roundStart,,,,,,) = fundingPot.getRoundGenericParameters(roundId);
+        vm.warp(roundStart + 1);
+
+        ILM_PC_FundingPot_v1.AccessCriteria memory accessCriteria =
+            _helper_createAccessCriteria(accessCriteriaEnum);
+
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                ILM_PC_FundingPot_v1
+                    .Module__LM_PC_FundingPot__RoundAlreadyStarted
+                    .selector
+            )
+        );
+        fundingPot.editAccessCriteriaForRound(
+            roundId, accessCriteriaId, accessCriteria
+        );
+    }
+
+    function testFuzzEditAccessCriteriaForRound(
+        uint8 accessCriteriaEnumOld,
+        uint8 accessCriteriaEnumNew
+    ) public {
+        vm.assume(accessCriteriaEnumOld >= 1 && accessCriteriaEnumOld <= 4);
+        vm.assume(
+            accessCriteriaEnumNew != accessCriteriaEnumOld
+                && accessCriteriaEnumNew >= 1 && accessCriteriaEnumNew <= 4
+        );
+
+        _helper_setupRoundWithAccessCriteria(accessCriteriaEnumOld);
+        uint64 roundId = fundingPot.getRoundCount();
+        uint8 accessCriteriaId = 0;
+
+        // Create and apply new access criteria
+        ILM_PC_FundingPot_v1.AccessCriteria memory newAccessCriteria =
+            _helper_createAccessCriteria(accessCriteriaEnumNew);
+
+        fundingPot.editAccessCriteriaForRound(
+            roundId, accessCriteriaId, newAccessCriteria
+        );
+
+        // Verify the access criteria was updated
+        (
+            bool isOpen,
+            address nftContract,
+            bytes32 merkleRoot,
+            address[] memory allowedAddresses
+        ) = fundingPot.getRoundAccessCriteria(roundId, accessCriteriaId);
+
+        assertEq(isOpen, accessCriteriaEnumNew == 1);
+        assertEq(nftContract, newAccessCriteria.nftContract);
+        assertEq(merkleRoot, newAccessCriteria.merkleRoot);
+        assertEq(allowedAddresses, newAccessCriteria.allowedAddresses);
+    }
 
     // -------------------------------------------------------------------------
     // Test: Internal Functions
+
+    // -------------------------------------------------------------------------
+    // Helper Functions
+
+    // @notice Creates a default funding round
+    function _helper_createDefaultFundingRound()
```

Comment:
Now that you have added fuzz tests, I still think this test would be easiert to read if the default values were stored on top of the file. Could we change this?

Reply:
Yup just added this to the top, I basically created a set of default values and then a set of edited values. Though I do think for some tests like creating a round these values should be fuzzed as we should be testing a bunch of different combinations of possible rounds to make sure there aren't an edge cases that break the creation of a round, however I agree that having these in the set up makes the most sense so Im changing that for now.